webcomponents-loader.js

```diff
@@ -10,15 +10,73 @@
 
 (function() {
   'use strict';
-  // global for (1) existence means `WebComponentsReady` will fire,
-  // (2) WebComponents.ready == true means event has fired.
-  window.WebComponents = window.WebComponents || {};
+
+  var polyfillsLoaded = false;
```

Comment:
Let's add some comments explaining the basic flow here.

Reply:
Done